static/app/views/insights/common/queries/useReleases.tsx

```diff
@@ -30,7 +39,8 @@ export function useReleases(searchTerm?: string) {
           per_page: 50,
           environment: environments,
           query: searchTerm,
-          sort: 'date',
+          sort: activeSort === 'number_events' ? 'date' : activeSort,
+          flatten: activeSort === 'date' || activeSort === 'number_events' ? 0 : 1,
```

Comment:
depending on the selected sortBy option, this is needed or we get an error from the backend. A similar logic can be found [here](https://github.com/getsentry/sentry/blob/5078825918dfc42a0fd3cccbaa3643fdbc776d15/static/app/views/releases/list/index.tsx#L106)

Reply:
Worth adding this as a code comment :)